kombu/transport/redis_cluster.py

```diff
@@ -237,7 +239,7 @@ def get_connection(cls, host, port):
     def create_connection(cls, host, port):
         params = {'skip_full_coverage_check': True, 'host': host, 'port': port}
 
-        return redis.RedisCluster(**params)
+        return RedisCluster(**params)
```

Comment:
I think we should hand over connection pool to its constructor and it is important in producer side